File: SoftUni Lessons/Python Development/Python Basics April 2019/Past Exams and Problems/3 - 3 and 4 November 2018/0 - My Exam/02. Movie Day .py
time_for_movie = int(input())
count_scenes = int(input())
time_one_scene = int(input())

preparation = time_for_movie * 0.15
time_all_scenes = count_scenes * time_one_scene
needed_time = preparation + time_all_scenes

if time_for_movie >= needed_time:
    print(f"You managed to finish the movie on time! You have {round(time_for_movie - needed_time)} minutes left!")
else:
    print(f"Time is up! To complete the movie you need {round(needed_time - time_for_movie)} minutes.")

File: SoftUni Lessons/JavaScript Development/JavaScript Basics March 2020/21. МИНАЛИ ИЗПИТИ/04. Online Exam - 15 and 16 June 2019/02. Movie Day.js
function movieDay(params) {
    let timePictures = Number(params.shift());
    let countScenes = Number(params.shift());
    let timeScene = Number(params.shift());

    let actualTime = (timePictures * 0.15) + (countScenes * timeScene); 
    
    if (actualTime <= timePictures) {
        console.log(`You managed to finish the movie on time! You have ${Math.round(timePictures - actualTime)} minutes left!`);
    } else {
        console.log(`Time is up! To complete the movie you need ${Math.round(actualTime - timePictures)} minutes.`);
    }
}


movieDay(['120', '10', '11']);  // Expected Output:
// Time is up! To complete the movie you need 8 minutes.

movieDay(['60', '15', '3']);  // Expected Output:
// Time is up! To complete the movie you need 8 minutes.
